From 8c707dd7af988ea29b569f42a8c462c86d65817b Mon Sep 17 00:00:00 2001
From: Franklin S. Cooper Jr <fcooper@ti.com>
Date: Fri, 10 May 2013 10:43:36 -0500
Subject: [PATCH] wpagui: Fix was not declared in this scope when using gcc 4.7 based toolchain

* Adding missing header to fix the following errors:
    error: 'getopt' was not declared in this scope
    error: 'optarg' was not declared in this scope

Signed-off-by: Franklin S. Cooper Jr <fcooper@ti.com>
---
 wpagui.cpp |    5 +----
 1 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/wpagui.cpp b/wpagui.cpp
index 97bf5ac..4b5ec23 100644
--- a/wpagui.cpp
+++ b/wpagui.cpp
@@ -12,16 +12,13 @@
  * See README and COPYING for more details.
  */
 
-#ifdef __MINGW32__
-/* Need to get getopt() */
-#include <unistd.h>
-#endif
 
 #ifdef CONFIG_NATIVE_WINDOWS
 #include <windows.h>
 #endif /* CONFIG_NATIVE_WINDOWS */
 
 #include <cstdio>
+#include <unistd.h>
 #include <QMessageBox>
 #include <QCloseEvent>
 #include <QImageReader>
-- 
1.7.0.4

